spec/ransack/adapters/active_record/base_spec.rb

```diff
@@ -145,6 +145,9 @@ def self.sane_adapter?
           end
 
           it "should function correctly with a multi-parameter attribute" do
+            ::ActiveRecord::Base.default_timezone = :utc
+            Time.zone = 'UTC'
```

Comment:
Normally, if just temporary, a `Time.use_zone` block around the related code might have been better - here also to avoid unexpected behaviour other places in the test suite.